src/helpers/measureTitleHelper.ts

```diff
@@ -272,3 +272,33 @@ export function fillMissingTitles(
     const measureTitleProps = buildMeasureTitles(measureBucketItems, locale, maxArithmeticMeasureTitleLength);
     return updateVisualizationObjectTitles(mdObject, measureTitleProps);
 }
+
+function removeTitleFromMeasures(bucket: IBucket): IBucket {
+    return {
+        ...bucket,
+        items: bucket.items.map(bucketItem => {
```

Comment:
Please add type for `bucketItem` and return type

Reply:
Fixed!